colors/scoreboard.json.example

```diff
@@ -124,7 +124,7 @@
       "g": 255,
       "b": 255
     },
-    "strikeout": {
+    "play_result": {
```

Comment:
I think I'd want Ks and hits to be two different colors, if we could allow them to be split

Reply:
Sounds good to me. Red for K’s and yellow for everything else? Or did you have something else in mind?

Reply:
Yeah that makes sense as a default. The important thing is letting them be customized

Reply:
Sounds good, it's implemented now